csharp/src/SeedLang.Ast/Executor.cs

```diff
@@ -21,6 +23,10 @@ public sealed class Executor : AstWalker {
     // The visualizer center to observe AST execution events and dispatch them to the registered
     // visualizers.
     private readonly VisualizerCenter _visualizerCenter;
+
+    // The dictionary to store variable names and current values of global variables.
+    private readonly Dictionary<string, BaseValue> _globals = new Dictionary<string, BaseValue>();
```

Comment:
Should we have a common class hierarchy for symbol tables, located in SeedLang.Runtime? Will the symbol table classes be sharable between the AST executor and the Bytecode executor?

Or, it's also okay to keep the current simple dictionary and leave some TODO comments here.

Reply:
We may need to define symbol table classes to handle the global variables in AST component and Bytecode component. But their behavior might be quite different, because the symbol table class in Bytecode component might be used in compile time, instead of runtime. It's hard to decide if they will have a common base class located in SeedLang.Runtime now.
Anyway, I will add a TODO comment to remind us define the symbol table class in the future.